### ISSUE TITLE
how to get train_A and train_inst from real images 

### ISSUE THREAD

**Issue by jiameij** [NVIDIA/vid2vid#61]
thanks for releasing the code.
in readme, dataset->Cityscapes

> We apply a pre-trained segmentation algorithm to get the corresponding semantic maps (train_A) and instance maps (train_inst).

which segmentation algorithm do you use? 
thank you !

**Comment by tcwang0509** [comment]
For segmentation, we used an internal algorithm that we plan to release in a couple of months. For now, any algorithm on the Cityscapes [benchmark](https://www.cityscapes-dataset.com/benchmarks/) that has their code released should work; our method is pretty robust to the segmentation masks.
For instances, we just used Mask RCNN. This is actually not very important, as the network wasn't trained on Cityscapes and doesn't perform really well.

**Comment by forfun1012** [comment]
They mentioned in the paper, as Deeplab for pixel segment, and Mask R-CNN for instances.
